src/result_object/helpers/fail/index.js

```diff
@@ -1,7 +1,8 @@
 // @flow
 import PassableResponse from '../../index';
 
-export const WARN: string = 'warn';
+export const WARN: Severity = 'warn';
+export const FAIL: Severity = 'fail';
 
 type CountName = 'failCount' | 'warnCount';
 type ObjectName = 'validationErrors' | 'validationWarnings';
```

Comment:
I don't understand why you enrich the `this` and return it instead of return a new object, can we sit together?

Reply:
is `this` the res of Passable??

Reply:
Good point. Moving fail into the result object class.